Formula/haruhi-dl.rb

```diff
@@ -0,0 +1,23 @@
+class HaruhiDl < Formula
+  desc "Fork of youtube-dl, focused on bringing a fast, steady stream of updates"
+  homepage "https://haruhi.download"
+  url "https://git.sakamoto.pl/laudompat/haruhi-dl/-/archive/v2021.03.21/haruhi-dl-v2021.03.21.tar.gz"
+  sha256 "8b7304635ecf7b6c7f36de9ae15433827e9690304f8f6e6f495805b9c820278e"
+  license "LGPL-3.0-or-later"
+
+  depends_on "pandoc" => :build
+  depends_on "python@3.9"
+
+  def install
+    system "make", "haruhi-dl", "haruhi-dl.1", "haruhi-dl.bash-completion", "haruhi-dl.zsh", "haruhi-dl.fish"
+    bin.install "haruhi-dl"
+    man1.install "haruhi-dl.1"
+    bash_completion.install "haruhi-dl.bash-completion"
+    fish_completion.install "haruhi-dl.fish"
+    zsh_completion.install "haruhi-dl.zsh"
+  end
+
+  test do
+    system "#{bin}/haruhi-dl", "--simulate", "https://www.youtube.com/watch?v=k8ozVkIkr-g"
```

Comment:
Is there some output we can check here? And this seems to be copyrighted, please use something with a CC license to test.

Reply:
sure, changed